frontend/src/storybook.test.tsx

```diff
@@ -16,6 +16,8 @@ class OurConverter extends Stories2SnapsConverter {
   }
 }
 
+jest.mock('@mui/utils/useId', () => jest.fn().mockReturnValue('mui-test-id'));
```

Comment:
Can you please add this to its own commit?

Reply:
same as the other one, without this change it wont pass CI

Reply:
It can be added as a separate earlier commit to the one it's in now. It would be good to put this in the same commit that upgrades the mui version.